packages/component/src/BasicTranscript.tsx

```diff
@@ -508,11 +509,16 @@ const InternalTranscript = forwardRef<HTMLDivElement, InternalTranscriptProps>(
       useCallback(() => focusByActivityKey(undefined), [focusByActivityKey])
     );
 
+    const [{ connectivitystatus: connectivityStatus }] = useDebouncedNotifications();
+    const statusMessage = connectivityStatus?.message;
+    const isWaiting = !statusMessage || statusMessage === 'connecting' || statusMessage === 'reconnecting';
+
     return (
       <div
         // Although Android TalkBack 12.1 does not support `aria-activedescendant`, when used, it become buggy and will narrate content twice.
         // We are disabling `aria-activedescendant` for Android. See <ActivityRow> for details.
         aria-activedescendant={android ? undefined : activeDescendantId}
+        aria-busy={isWaiting}
```

Comment:
Do we need `aria-busy` for `role="group"`? Or just the `role="feed"`?

If we only need it for `role="feed"`, then we can put the `useDebouncedNotifications()` inside `<InternalTranscriptScrollable>`.

Reply:
I've looked it up and `aria-busy` doesn't specify which roles it should or shouldn't be attached to. Since this div covers the whole transcript, I think it does make sense to mark it as busy when chat is loading.